app/models/dataset_record/darwin_core/occurrence.rb

```diff
@@ -236,11 +236,41 @@ def import(dwc_data_attributes = {})
             identifier_object: collecting_event
           }.merge!(identifier_attributes)) unless identifier_attributes.nil?
 
-          Georeference::VerbatimData.create!({
-            collecting_event: collecting_event,
-            error_radius: get_field_value("coordinateUncertaintyInMeters"),
-            no_cached: true
-          }.merge(attributes[:georeference])) if collecting_event.verbatim_latitude && collecting_event.verbatim_longitude
+
+          has_shape = self.import_dataset.metadata['import_settings']['require_geographic_area_has_shape'] if self.import_dataset.metadata['import_settings'].include? 'require_geographic_area_has_shape'
+          data_origin = self.import_dataset.metadata['import_settings']['geographic_area_data_origin'] if self.import_dataset.metadata['import_settings'].include? 'geographic_area_data_origin'
+          if collecting_event.verbatim_latitude && collecting_event.verbatim_longitude
+            Georeference::VerbatimData.create!({
+              collecting_event: collecting_event,
+              error_radius: get_field_value("coordinateUncertaintyInMeters"),
+              no_cached: true
+            }.merge(attributes[:georeference]))
+            location_levels = [collecting_event[:cached_level2_geographic_name], collecting_event[:cached_level1_geographic_name], collecting_event[:cached_level0_geographic_name]].compact
+          else
```

Comment:
`else` migth be OK, but I think we should try in all cases if the option is set to try?

Reply:
I could be misunderstanding you, but it does attempt to setup the GA either way. It just prefers using the latitude/longitude if it is provided over trying to use the county, state/province, country fields.

Reply:
You would be surprised, but people make mistakes in lat/long as often as in country names.